core/src/test/java/org/apache/brooklyn/core/entity/PolicyRegistrationTest.java

```diff
@@ -79,7 +80,7 @@ public void setUp() throws Exception {
     
     @Test
     public void testGetPoliciesIsInitiallyEmpty() {
-        assertEquals(entity.getPolicies(), ImmutableList.of());
+        assertEquals(ImmutableList.copyOf(entity.policies()), ImmutableList.of());
```

Comment:
Maybe suggest Guava's `Iterables.elementsEqual()` here (and elsewhere) but no strong feelings.